react.js

```diff
@@ -1,30 +1,43 @@
 import {writable, get} from 'svelte/store';
-import {beforeUpdate} from 'svelte';
+import {afterUpdate} from 'svelte';
 
 let currentStore;
 let i = 0;
+let effects = [];
+let layoutEffects = [];
 
 export function wrap(fn) {
   let store = writable([]);
-  currentStore = store;
-  i = 0;
-
-  let res = fn();
-
   let stores = {};
-  for (let key in res) {
-    stores[key] = writable(res[key]);
-  }
+  
+  let _effects = effects;
+  let _layoutEffects = layoutEffects;
+
+  afterUpdate(() => {
+    _effects.forEach(fn => fn());
+  });
 
   store.subscribe(function () {
```

Comment:
`store.subscribe` callback will be called synchronously for the 1st time upon subscribing. that's how `get(store)` gets its value too.